src/test/java/uk/gov/hmcts/juror/api/moj/service/CompleteServiceServiceImplTest.java

```diff
@@ -177,43 +220,33 @@ private void validateCompleteWasSuccess(JurorPool jurorPool, Juror juror, LocalD
         }
 
         @Test
-        void negativeNotInRespondedStatus() {
-            LocalDate localDate = LocalDate.of(2023, 11, 22);
-
-            final String firstName = RandomStringUtils.randomAlphabetic(20);
-            final String lastName = RandomStringUtils.randomAlphabetic(20);
-            final int poolStatus = IJurorStatus.JUROR;
-
-            Juror juror = mock(Juror.class);
-            JurorPool jurorPool = mock(JurorPool.class);
-
-            createJurorPoolMock(TestConstants.VALID_POOL_NUMBER, poolStatus, TestConstants.VALID_JUROR_NUMBER,
-                firstName, lastName, jurorPool, juror);
-
-            CompleteServiceJurorNumberListDto completeServiceJurorNumberListDto =
-                createCompleteServiceJurorNumberListDto(localDate, TestConstants.VALID_JUROR_NUMBER);
-
-            MojException.BusinessRuleViolation exception = assertThrows(MojException.BusinessRuleViolation.class,
-                () -> completeServiceService
-                    .completeService(TestConstants.VALID_POOL_NUMBER, completeServiceJurorNumberListDto),
-                "Expected exception to be thrown when juror pool not in resolved status");
-
-            assertEquals("Juror number " + TestConstants.VALID_JUROR_NUMBER
-                    + " is not in a valid state to complete service",
-                exception.getMessage(),
-                "Expected exception message to be " + "Juror number " + TestConstants.VALID_JUROR_NUMBER
-                    + " is not in a valid state to complete service");
-
+        @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+        void negativeNotInTransferredStatus() {
+            testInvalidStatus(IJurorStatus.TRANSFERRED);
+        }
 
-            verify(jurorPoolRepository, times(1))
-                .findByJurorJurorNumberAndPoolPoolNumber(TestConstants.VALID_JUROR_NUMBER,
-                    TestConstants.VALID_POOL_NUMBER);
+        @Test
+        @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+        void negativeNotInFailedToAttendStatus() {
+            testInvalidStatus(IJurorStatus.FAILED_TO_ATTEND);
+        }
 
-            verify(jurorPool, times(1)).getStatus();
+        @Test
+        @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+        void negativeNotInSummonedStatus() {
+            testInvalidStatus(IJurorStatus.SUMMONED);
+        }
 
+        @Test
+        @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+        void samplePositiveCompletedStatus() {
+            testValidStatus(IJurorStatus.COMPLETED);
+        }
 
-            verifyNoMoreInteractions(jurorPoolRepository, jurorPoolRepository, jurorRepository, jurorHistoryService,
-                juror, jurorPool);
+        @Test
+        @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
+        void samplePositiveExcusedStatus() {
+            testValidStatus(IJurorStatus.EXCUSED);
```

Comment:
If you rename testValidStatus to assertValidStatus you wont need to suppress JUnitTestsShouldIncludeAssert

Reply:
Fixed